src/yachtzee.ts

```diff
@@ -1117,6 +1117,9 @@ function _yachtzeeChain(): void {
   let turncount = myTurncount();
   yachtzeePotionSetup(Math.min(jellyTurns, fishyTurns));
   cliExecute(`closet take ${myClosetMeat()} meat`);
+  if (haveEffect($effect`Beaten Up`)) {
+    cliExecute("hottub");
```

Comment:
if you aren't able to hottub, we should uneffect beaten up in some other way, using libram's uneffect()

Reply:
Resolved in latest commit